src/gui/mzroll/ligandwidget.cpp

```diff
@@ -102,7 +102,7 @@ LigandWidget::LigandWidget(MainWindow* mw)
   m_manager = new QNetworkAccessManager(this);
   connect(m_manager,SIGNAL(finished(QNetworkReply*)),this,SLOT(readRemoteData(QNetworkReply*)));
 
-  QDirIterator itr(":/databases/");
+  QDirIterator itr("src/gui/mzroll/databases/");
```

Comment:
This is the relative path for mzroll, but now when database class is moved to libmaven. It can no longer read files through relative path. Hence, given the full path.

Reply:
But we are still calling the directory iterator from `LigandWidget` which should have access to resource paths. Once you are iterating over the files within that directory, then is it not possible to provide the embedded path somehow to DB? If not, you should still figure out a way to send the contents of the resource file (instead of the file-path) to DB and then it can parse that byte-stream or something. Because using `src/...` path will not work once El-MAVEN is packaged into an installer and sent out to users. The only way to access files is to embed them in the binary itself - which is what the `:/resource` files are (whether it is databases, images, or whatever).

Reply:
ohh! Okayy will make this change.